backend/src/admin/admin.service.ts

```diff
@@ -31,4 +26,17 @@ export class AdminService {
         details.email = details.email.toLowerCase();
```

Comment:
Not really part of this PR, but just noticed. In my opinion `createAdmin` would be better name for the function. This function could also be removed by using a Pipe that transformes the email to lowercase in the controller.

Reply:
As an alternative, this is now handled by TpyeORM on the "BeforeInsert()" hook.